mobify/platform-scaffold
# [Integration Manager] Switch to using the notifications Redux branch from the SDK

This PR removes the notifications storage from the `app/ui` store branch, instead installing the SDK port of the notification storage at the root of the store. 

 **JIRA**: N/A
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/629

## Changes
- Remove the old notification system
- Use the new one

## How to test-drive this PR
- Preview the site after an `npm install`
- Cause a notification to occur (the easiest way I have found is by setting the page to offline and then attempting to add to cart)
- See that the notification appears, and dismisses when the close button is clicked
- Cause another notification to occur
- See that it goes away on navigating to a new page.
